db_stress_tool/db_stress_test_base.cc

```diff
@@ -3398,6 +3398,27 @@ void StressTest::Reopen(ThreadState* thread) {
   }
   column_families_.clear();
 
+  // TODO(hx235):  Currently reopen does not restore expected state
+  // with potential data loss in mind like the first open before
+  // crash-recovery verification does. Therefore it always expects no data loss
+  // and we should ensure no data loss in testing.
```

Comment:
My understanding after reading this several times is the goal of the TODO is to eliminate the `FlushWAL(true /* sync */)`/`SyncWAL()` below. If that's the case can we say that's the goal?

Reply:
Yeah this was not good written TODO - let me fix it.

Reply:
Fixed. The previous one is more of a comment than a TODO.